### ISSUE TITLE
[No buffer space available] error after a commit in zbalance_ipc

### ISSUE THREAD

**Issue by gianogli** [ntop/PF_RING#667]
After this [commit](https://github.com/ntop/PF_RING/commit/38633d351c90e9e32a352ab20dd18ef26b1bcbe3#diff-73016be3be9cd3b77f9dc5f1b2dd0a4674c522b48ace008a96e0c00bfe001b61) the zbalance_ipc is not usable with a number of application instances (-n) greater than 2:
```
# ./zbalance_ipc -i zc:eth6@0,zc:eth8@0 -c10 -n3 -m3 -g8 -p 
22/Oct/2020 11:38:56 [zbalance_ipc.c:1270] ERROR: pfring_zc_create_queue_pool_pair error [No buffer space available]
```

Reverting the commit I'm able to run the same command or also
```
# ./zbalance_ipc -i zc:eth6@0,zc:eth8@0 -c10 -n10 -m3 -g8 -p 
```
without any errors.

Could you solve the issue?

I'm using PF_RING branch 7.8.0-stable.

Thanks... :-)

**Comment by cardigliano** [comment]
Fixed in cd20ff2693c9cf91e0d1f2bf98a6143a9a2c1ae7 
Please verify

**Comment by gianogli** [comment]
Verified. It works.

Thanks... :-)
